Should shorten URLs in newsletter introduction only if the workspace setting on

caiosba opened the pull request:
## Description

Should shorten URLs in newsletter introduction only if the workspace setting on.

Fixes CV2-3502.

## How has this been tested?

![Captura de tela de 2023-07-21 17-47-50](https://github.com/meedan/check-api/assets/117518/a97e977e-9797-4c29-8477-5e792c9bc962)

## Checklist

- [x] I have performed a self-review of my own code
- [ ] I have added unit and feature tests, if the PR implements a new feature or otherwise would benefit from additional testing
- [ ] I have added regression tests, if the PR fixes a bug
- [ ] I have added logging, exception reporting, and custom tracing with any additional information required for debugging
- [ ] I have commented my code in hard-to-understand areas, if any
- [ ] I have made needed changes to the README
- [x] My changes generate no new warnings
- [ ] If I added a third party module, I included a rationale for doing so and followed our current [guidelines](https://meedan.atlassian.net/wiki/spaces/ENG/overview#Choose-the-%E2%80%9Cright%E2%80%9D-3rd-party-module)

hartsick reviewed (approved):
Looks good! Know this is a quick fix for an issue in production, but wondering if there's a small test we could add in to reinforce this behavior after merge.

caiosba commented:
> Looks good! Know this is a quick fix for an issue in production, but wondering if there's a small test we could add in to reinforce this behavior after merge.

Good catch, thanks! I added it now - I also added another test that was missing.
